addons/website_slides/static/src/js/slides_course_fullscreen_player.js

```diff
@@ -618,6 +616,9 @@ odoo.define('website_slides.fullscreen', function (require) {
             return this._fetchSlideContent().then(function() { // render content
                 var websiteName = document.title.split(" | ")[1]; // get the website name from title
                 document.title =  (websiteName) ? slide.name + ' | ' + websiteName : slide.name;
+                if  (config.device.size_class < config.device.SIZES.MD) {
+                    self._toggleSidebar();
+                }
```

Comment:
Can you add a small explanation of what this does in the method docstring?
I'm guessing it addresses that spec point:

> Open the sidebar and change slide. The sidebar should automically be closed when the slide changes

Reply:
Done. Also cleaned up the English a bit in the docstring that already existed.